funcbench/env.go

```diff
@@ -62,25 +58,14 @@ func newLocalEnv(e environment) (Environment, error) {
 	if err != nil {
 		return nil, err
 	}
-
+	e.logger.Println("[Local Mode]")
```

Comment:
Same here, would nice if you can format message to be as you want and just print once